api-docs/openapi-diff.ts

```diff
@@ -0,0 +1,37 @@
+/* global process */
+/* eslint import/no-nodejs-modules:0 */
+/* eslint import/no-unresolved:0 */
+
+const fs = require('fs');
+const yaml = require('js-yaml');
+const jsonDiff = require('json-diff');
+const https = require('https');
+
+async function main() {
+  const request = new Promise((resolve, reject) =>
+    https.get(
+      `https://raw.githubusercontent.com/getsentry/sentry-api-schema/main/openapi-derefed.json`,
+      res => {
+        res.setEncoding('utf8');
+        let rawData = '';
+        res.on('data', chunk => (rawData += chunk));
+        res.on('end', () => {
+          try {
+            resolve(JSON.parse(rawData));
+          } catch (e) {
+            reject(e.message);
+          }
+        });
+      }
+    )
+  );
+
+  const openApiData = await request;
+  const readFile = fs.readFileSync('tests/apidocs/openapi-derefed.json', 'utf8');
```

Comment:
Ah yea, `fs.readfile` takes a callback and rather than promisifying it i figured it would just be simpler to await the https request, then just read the file 🤷 

happy to go the other route if that's preferred